### DISCUSSION TITLE
The initial number of bars in the project is only 32, how can I add extra bars within the project?

### DISCUSSION THREAD

**Discussion by NYMHY** [stakira/OpenUtau#923] [Q&A]
The only way i can find is to expand a new part.

**Comment by oxygen-dioxide** [comment]
Yes, you have to make a new part and expand it
OpenUtau don't store the "total number of bars in a project" inside ustx file. It just shows a few bars after the end of the part that ends the last. Just like you can't create a word document with 100 empty pages. You have to input "Enter"s to do so.
